src/components/Advanced.js

```diff
@@ -1,269 +1,349 @@
-import React from 'react';
+import React from "react";
 import { Scaler } from "dapparatus";
 import { CopyToClipboard } from "react-copy-to-clipboard";
-import i18n from '../i18n';
-import {
-  Input,
-  QR as QRCode,
-  Text,
-  Select,
-  Flex,
-  Checkbox
-} from 'rimble-ui'
-import { PrimaryButton, BorderButton } from '../components/Buttons'
-import getConfig from '../config'
+import i18n from "../i18n";
+import { Input, QR as QRCode, Text, Select, Flex, Checkbox } from "rimble-ui";
+import { PrimaryButton, BorderButton } from "../components/Buttons";
+import getConfig from "../config";
 import { getStoredValue, storeValues } from "../services/localStorage";
 
-const { CURRENCY } = getConfig()
+const { CURRENCY } = getConfig();
 
 export default class Advanced extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      privateKeyQr:false,
-      seedPhraseHidden:true,
-      privateKeyHidden:true,
-      currency: '',
-      expertMode:false
-    }
-    this.updateCurrency = this.updateCurrency.bind(this)
+      privateKeyQr: false,
+      seedPhraseHidden: true,
+      privateKeyHidden: true,
+      currency: "",
+      expertMode: false
+    };
+    this.updateCurrency = this.updateCurrency.bind(this);
   }
 
   componentDidMount() {
     const { address } = this.props;
-    const currency = getStoredValue('currency', address) || CURRENCY.DEFAULT_CURRENCY;
-    const expertMode = getStoredValue("expertMode") === "true"
-      // Right now "expertMode" is enabled by default. To disable it by default, remove the following line.
-      || getStoredValue("expertMode") === null;
-    this.setState({ currency, expertMode })
+    const currency =
```

Comment:
Single line was looking better here :P

Reply:
Did you prettified the whole file? :)

Reply:
Oh :poop:, yeah I might have forgot to disable the prettifier here :sweat:.